### PR TITLE
Add clang tidy

### PR DESCRIPTION
**Pull request by Changochen**

### CODE REVIEW

File: srcs/internal/client/client_mysql.cc [hunk 1]
```diff
@@ -55,7 +55,8 @@ ExecutionStatus MySQLClient::execute(const char *query, size_t size) {
     return kServerCrash;
   }
 
-  auto status = clean_up_connection(*connection);
+  // auto status = clean_up_connection(*connection);
```
Comment (zr950624) [inline srcs/internal/client/client_mysql.cc:58]: If we do not need status, we can just remove this line.

Comment (Changochen) [reply srcs/internal/client/client_mysql.cc]: Change to a `TODO`.

File: srcs/internal/client/client_mysql.cc [hunk 2]
```diff
@@ -117,7 +118,8 @@ ExecutionStatus MySQLClient::execute_query(char *cmd) {
   std::cerr << "Server response: " << server_response << std::endl;
   std::cerr << "Server error: " << mysql_errno(&m_) << std::endl;
 
-  auto correctness = clean_up_connection(m_);
+  clean_up_connection(m_);
+  // auto correctness = clean_up_connection(m_);
```
Comment (zr950624) [inline srcs/internal/client/client_mysql.cc:122]: How about using TODO for all comments if we need further work on them? as well as line 172 and 209.

Comment (Changochen) [reply srcs/internal/client/client_mysql.cc:122]: Sure. Done.

**Comment by zr950624** [comment]
LGTM

**Review by zr950624 — APPROVED** [review]

**Pull request merged**